lms/djangoapps/courseware/testutils.py

```diff
@@ -31,7 +31,10 @@ class RenderXBlockTestMixin(object):
         '<footer id="footer-openedx"',
         '<div class="window-wrap"',
         '<div class="preview-menu"',
-        '<div class="container"'
+        '<div class="container"',
+        # Although bookmarks were removed from all chromeless views, it is LTI
+        # specifically that must never include them.
+        '<div class="bookmark-button-wrapper"',
```

Comment:
@nasthagiri: Here is an attempt at the simplified test for dropping bookmarks for chromeless/LTI.  Original PR is here: https://github.com/edx/edx-platform/pull/13820

Pros:
- This test code is much simpler.

Cons:
- Test is now vertical block specific.
- This drops the leaf node (html block) test.

Other comments:
- The original PR code added a sequential as well in the test course.  However, the sequential HTML-escapes all child HTML when in the chrome view, but not in the chromeless view.  Although you could add the [HTML-escaped version of the bookmark HTML here](https://github.com/edx/edx-platform/pull/14174/commits/2e5ee11ca477ffc446065a4178be423d74eb19f5), the negative test would pass even if it included the bookmark HTML, because it wouldn't contain the HTML-escaped version of the bookmark.  Could this be indicative of an issue with sequentials in chromeless view?  I'm not sure why the contents of a sequential are HTML-escaped, which you can see if you search for `bookmark-button-wrapper` when viewing Page Source of any courseware page.

Let me know what you think.

FYI: @dmitry-viskov